elements/lisk-validator/src/formats.ts

```diff
@@ -103,23 +92,9 @@ export const signedPublicKey = (data: string): boolean => {
 	}
 };
 
-export const transferAmount = isValidTransferAmount;
-
 export const transferData = (data: string): boolean =>
 	!isNullCharacterIncluded(data) && isValidTransferData(data);
 
-export const int64 = (data: string): boolean =>
-	isNumberString(data) && isInt64(BigInt(data));
-
-export const uint64 = (data: string): boolean =>
-	isNumberString(data) && isUint64(BigInt(data));
-
-export const uint32 = (data: string): boolean =>
-	isNumberString(data) && isUint32(BigInt(data));
-
-export const int32 = (data: string): boolean =>
-	isNumberString(data) && isInt32(BigInt(data));
-
```

Comment:
Why to remove these formats? With this change we can't specify these values as  `format` in the JSON schema.

Reply:
I thought `format` is now only for strings? also, issue says, `Remove formats which are replaced by the new validation for these dataTypes` @shuse2 ?

Reply:
@ishantiw Not its just for strings can be used for numbers as well. @shuse2 The HTTP Plugin we will develop will be relying on JSON data and if we need to validate any input on earlier stage we need to keep these formats.